ubuntu_desktop_installer/lib/l10n/intl_it.arb

```diff
@@ -0,0 +1,87 @@
+{
+  "@@last_modified": "2021-01-22T21:23:53.258261",
+  "Ubuntu Desktop Installer": "Ubuntu Desktop Installer",
+  "@Ubuntu Desktop Installer": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Go Back": "Torna indietro",
+  "@Go Back": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Continue": "Continua",
+  "@Continue": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Restart": "Riavvia",
+  "@Restart": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Try or install": "Prova o installa",
+  "@Try or install": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Repair installation": "Ripara installazione",
+  "@Repair installation": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Repairing will reinstall all installed software without touching documents or settings.": "La riparazione reinstallerá tutti i programmi senza toccare alcun documento o impostazione",
+  "@Repairing will reinstall all installed software without touching documents or settings.": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Try Ubuntu": "Prova Ubuntu",
+  "@Try Ubuntu": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "You can try Ubuntu without making any changes to your computer.": "Puoi provare Ubuntu senza apportare alcuna modifica al tuo computer",
```

Comment:
There should probably be a full stop at the end of the sentence, for consistency.